### PR TITLE
Feat(eos_cli_config_gen): add hardware flow tracking standard record format support

### LINKED ISSUE #3650: Feat(eos_cli_config_gen): add hardware flow tracking standard record format support
### Enhancement summary

In order to stream IPFIX-based flow tracking data to CloudVision, EOS must be configured with `record format ipfix standard timestamps counters` under `flow tracking hardware`.  This command is not currently exposed through eos_cli_config_gen.

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

CCS platforms (which support hardware flow tracking) showing traffic flow data in CloudVision.

### Describe the solution you would like

Add a boolean `record_format_ipfix_standard_timestamps_counters` (or similar) under `flow_tracking: hardware`

### Describe alternatives you have considered

`raw_eos_cli` at the "root" device level is the only current alternative I've found.  If `eos_cli` could be supported under `flow tracking: hardware`, this might be an alternative.

### Additional context

EOS documentation: https://www.arista.com/en/support/toi/eos-4-23-1f/14409-hardware-flow-tracking-with-ipfix-export#:~:text=To%20configure%20standard%20IPFIX%20information%20elements%20for%20timestamps%20and%20counters%3A

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by laxmikantchintakindi**
## Change Summary

Added support for hardware flow tracking standard record.

## Related Issue(s)

Fixes #3650

## Component(s) name

`arista.avd.eos_cli_config_gen

## Proposed changes
Added a key under flow-tracking:hardware. The value is a boolean.

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Review by ClausHolbechArista — DISMISSED** [review]
Please update the documentation template as well.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/flow_tracking.schema.yml [hunk 1]
```diff
@@ -65,3 +65,6 @@ keys:
       hardware:
         type: dict
         $ref: "eos_cli_config_gen#/$defs/flow_tracking"
+        keys:
+          record_format_ipfix_standard_timestamps_counters:
```
Comment (Shivani-gslab) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/flow_tracking.schema.yml]: There is two options inside record - `format` and `rewrite`.
So i think we should change the model to - 
```
record:
      type: dict
      keys:
          format_ipfix_standard_timestamps_counters:
                 type: bool
```

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/flow-tracking.j2 [hunk 2]
```diff
@@ -148,6 +148,16 @@
 {%             endfor %}
 {%         endif %}
 {%     endif %}
+{%     if flow_tracking.hardware.record.format_ipfix_standard_timestamps_counters is arista.avd.defined(true) %}
```
Comment (Shivani-gslab) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/flow-tracking.j2]:
```suggestion
{%     if flow_tracking.hardware.record is arista.avd.defined(true) %}
```

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/flow-tracking.j2 [hunk 3]
```diff
@@ -148,6 +148,16 @@
...
+
+##### Records Summary
+
+| Record | Value |
+| ------ | ----- |
+{%         if flow_tracking.hardware.record.format_ipfix_standard_timestamps_counters is arista.avd.defined(true) %}
+| Record format ipfix standard timestamps counters | {{ flow_tracking.hardware.record.format_ipfix_standard_timestamps_counters }} |
```
Comment (Shivani-gslab) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/flow-tracking.j2]:
```suggestion
| Format ipfix standard timestamps counters | {{ flow_tracking.hardware.record.format_ipfix_standard_timestamps_counters }} |
```

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/flow-tracking.j2 [hunk 4]
```diff
@@ -116,6 +116,11 @@ flow tracking hardware
       flow table size {{ tracker.table_size }} entries
 {%         endif %}
 {%     endfor %}
+{%     if flow_tracking.hardware.record.format_ipfix_standard_timestamps_counters is arista.avd.defined(true) %}
+{%         if flow_tracking.hardware.record.format_ipfix_standard_timestamps_counters is arista.avd.defined(true) %}
+   record format ipfix standard timestamps counters
+{%         endif %}
+{%     endif %}
```
Comment (Shivani-gslab) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/flow-tracking.j2]:
```suggestion
{%     if flow_tracking.hardware.record.format_ipfix_standard_timestamps_counters is arista.avd.defined(true) %}
   record format ipfix standard timestamps counters
{%     endif %}
```

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/flow_tracking.schema.yml (hunk 1 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/flow_tracking.schema.yml]: Updated the schema as per suggestion.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/flow-tracking.j2 (hunk 2 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/flow-tracking.j2]: Corrected the variable in condition.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/flow-tracking.j2 (hunk 3 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/flow-tracking.j2]: Corrected this suggestion.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/flow-tracking.j2 (hunk 4 above)
Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/flow-tracking.j2]: Completed these changes.

**Review by Shivani-gslab — APPROVED** [review]
LGTM !

**Review by MaheshGSLAB — APPROVED** [review]
LGTM, generated config on lab and tested

**Review by sugetha24 — APPROVED** [review]
LGTM. 
Tested: Molecule test and documentation.

**Review by gmuloc — APPROVED** [review]
LGTM

**Pull request merged**